keyboards/handwired/bluepill/bluepill.h

```diff
@@ -0,0 +1,57 @@
+#ifndef KEYBOARDS_CHIBIOS_TEST_CHIBIOS_TEST_H_
```

Comment:
If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?

Reply:
That throws the error:
In file included from keyboards/handwired/bluepill/bluepill.c:1:0:
keyboards/handwired/bluepill/bluepill.h:40:29: error: backslash-newline at end of file [-Werror]
 #define LAYOUT_seventy_iso( \

cc1.exe: all warnings being treated as errors
 |
 |
 |
make[1]: *** [tmk_core/rules.mk:360: .build/obj_handwired_bluepill_bluepill70_default/keyboards/handwired/bluepill/bluepill.o] Error 1


Despite that I'm just commiting the other changes, I need to test everything, I give you a word when I have it tested and working because I have the controllers at a home under remodelling. At that moment it compiles only with the cortex-m3 firmware size warning